packages/clerk-js/src/core/modules/billing/namespace.ts

```diff
@@ -39,7 +39,7 @@ export class Billing implements BillingNamespace {
     const { for: forParam, ...safeParams } = params || {};
     const searchParams = { ...safeParams, payer_type: forParam === 'organization' ? 'org' : 'user' };
     return await BaseResource._fetch({
-      path: `/commerce/plans`,
+      path: `${Billing.#pathRoot}/plans`,
```

Comment:
🔧 Looks like this is calling an authenticated endpoint `/me/billing/plans` but the actual path is just `/billing/plans`